packages/legend-graph/src/graph-manager/protocol/pure/v1/transformation/pureGraph/to/helpers/V1_ValueSpecificationBuilderHelper.ts

```diff
@@ -766,11 +795,40 @@ function buildPropertyGraphFetchTree(
     processingContext.flushVariable('this');
     processingContext.pop();
   } else {
-    property = V1_getAppliedProperty(
-      parentClass,
-      undefined,
-      propertyGraphFetchTree.property,
-    );
+    //While building property graph fetch tree, the Class of root graph
+    //fetch tree is passed as the parentClass of property graph fetch tree.
+    //This Class contains the properties of property graph fetch trees.
```

Comment:
indent this better

Reply:
let's do something like


```suggestion
    // This Class contains the properties of property graph fetch trees.
    // ...
```